frontend/src/data/variables/BrfssProvider.ts

```diff
@@ -52,6 +53,7 @@ class BrfssProvider extends VariableProvider {
       breakdowns.demographicBreakdowns.race.enabled &&
       breakdowns.demographicBreakdowns.race.includeTotal
     ) {
+      // TODO totals
```

Comment:
What's the TODO here for specifically?

Reply:
oh oops I think this was not meant to stay- deleted it